firefox-ios/Client/Frontend/Browser/TabTrayButtonExtensions.swift

```diff
@@ -32,7 +32,9 @@ class PrivateModeButton: ToggleButton, PrivateModeUI {
     }
 
     func applyTheme(theme: Theme) {
-        tintColor = isSelected ? theme.colors.iconOnColor : theme.colors.iconPrimary
+        let color = theme.colors
```

Comment:
We get the colors from theme, so I think I'd be better to name the constant `colors`.